docs/drafts/how-to-channel-based-upgrades.md

```diff
@@ -0,0 +1,44 @@
+## How-to: Channel-Based Automatic Upgrades
+
+A "channel" is a package author defined stream of updates for an extension. A set of channels can be set in the Catalog source to restrict automatic updates to the set of versions defined in those channels.
+
+Example:
+
+```yaml
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterExtension
+metadata:
+  name: argocd
+spec:
+  installNamespace: argocd
```

Comment:
There was a recent [PR](https://github.com/operator-framework/operator-controller/pull/1200) that put the `installNamespace` and `serviceAccount` fields under an `install` root field.